Cython/Utility/ModuleSetupCode.c

```diff
@@ -908,7 +908,11 @@ class __Pyx_FakeReference {
                                                           Py_ssize_t nargs, PyObject *kwnames);
 #else
   #define __Pyx_PyCFunctionFast _PyCFunctionFast
```

Comment:
Should the define using `_PyCFunctionFast` also be updated in the same way?

Reply:
Oh right, fixed.